helion/_compiler/inductor_lowering_extra.py

```diff
@@ -0,0 +1,49 @@
+"""
+NOTE: This file is only designed to be imported within `patch_inductor_lowerings()` function.
```

Comment:
We shouldn't assume we can control when files get imported, and importing files shouldn't have side effects.

Let's refactor this to not rely on import order.